tests/test_activities.py

```diff
@@ -1,10 +1,30 @@
 import json
 
-from api.models import Activity
+from api.models import Activity, ActivityType
 from tests.base_test import BaseTestCase
 
 
 class ActivitiesTestCase(BaseTestCase):
     """Test activities endpoints."""
     pass
-        
+
+
+class ActivityTypesTestCase(BaseTestCase):
+    """Test activity types endpoints."""
+
+    def test_get_activity_types(self):
+        # add test activity types
+        self.hackathon.save()
+        self.tech_event.save()
+
+        response = self.client.get('api/v1/activity-types',
+                                    headers={"Authorization":self.test_token})
```

Comment:
You don't need to use this. There is a `self.header` in the `BaseTestCase` you can just do:
`headers=self.header`

Reply:
Missed this as well. Thanks!